app/models/grda_warehouse/us_census_api/town_agg.rb

```diff
@@ -0,0 +1,152 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# https://www.census.gov/content/dam/Census/data/developers/api-user-guide/api-guide.pdf
+# https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
+
+module GrdaWarehouse
+  module UsCensusApi
+    class TownAgg
```

Comment:
Looks okay to me. Were you just saving time by not updating the CoC one to be more generic?

Reply:
Didn't even think of it, but yes, time was  a constraint.